test/drenv/__main__.py

```diff
@@ -37,15 +38,26 @@ def main():
         env = envfile.load(f, name_prefix=args.name_prefix)
 
     func = globals()[CMD_PREFIX + args.command]
-    func(env)
+
+    if args.skip_tests and "cmd_start" in str(func.__name__):
+        func(env, 1)
+    else:
+        func(env)
 
 
-def cmd_start(env):
+def cmd_start(env, skiptest=0):
```

Comment:
would it make sense to use bool values instead of `0` here, (though sometimes, the outcome is the same)